src/main/java/com/laborete/LaboreteAPI/controller/UsersController.java

```diff
@@ -0,0 +1,29 @@
+package com.laborete.LaboreteAPI.controller;
+
+import com.laborete.LaboreteAPI.entity.Users;
+import com.laborete.LaboreteAPI.services.UsersService;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.UUID;
+
+@RestController()
+@RequestMapping("v1/users")
+public class UsersController {
+    private final UsersService usersService;
```

Comment:
Please define an interface - name it `UsersService` and the implemention should be named `UsersServiceImpl`